package-lock.json

```diff
@@ -1,8 +1,15952 @@
 {
```

Comment:
I'm finding this diff quite difficult to parse. What was the command that you ran to generate this change? Might be worth popping in the commit message.

Reply:
Have added to the commit message, but it was running `npm install` with the latest stable version of npm - I'm not sure why it changed so much, but it seems to do so every time I run `npm install` (and did on one of Israt's branches too) so thought it'd be good to update here at least. I kept it in its own commit (separate from the one that adds the Stylelint package) to make it a bit clearer that it's separate.